web-server/views/users/profile.jade

```diff
@@ -0,0 +1,31 @@
+extends ../layout
+
+block content
+
+  .row
+    .col-md-4
+      h1= user.name
+      ul.list-inline
+        li
+          a(href=user.profileUrl target="_blank") @#{user.handle}
+        li
+          a(href=user.echoProfileURL target="_blank") Echo
+        li
+          a(href="https://curriculum.learnersguild.org/phases/4/") Phase: 3 (7/8)
+      img.img-thumbnail.user-avatar.user-avatar-huge(src="#{user.avatarUrl}" alt="avatar")
+    .col-md-8.profile-content
+      h6
+        | Email: 
+        a(href="mailto:#{user.email}") #{user.email}
+      h6 Phone: #{user.phone}
+      h6 Join Date: #{renderDate(user.createdAt)}
+      h6 Updated: #{renderDate(user.updatedAt)}
+      h6 
+        | Learning Facilitator: 
+        a(href="https://learnersguild.slack.com/messages/@carlabagdonas" target="_blank") Carla
+      h6 Professional Days Remaining: 3
+      h6 Personal Days Remaining: 3
+      h6 Phase 4 Interview Date: 2017/09/01
```

Comment:
still working on this part?

Reply:
I'm not sure how to continue on this part. I don't have access to this information. Should I add those key/value pairs to the object and pass that into the template?

Reply:
yes just manually sub in that data i sent you in slack